jupyter-client.el

```diff
@@ -37,22 +37,23 @@
 (require 'jupyter-mime)
 (require 'jupyter-messages)
 
-(defcustom jupyter-display-short-eval-result-function #'message
```

Comment:
I would prefer the name to be `jupyter-eval-short-result-display-function`. I don't think there is enough context in `jupyter-short-result-function` to understand what it is doing.

Reply:
That's what I changed it to.

Reply:
Duh. Sorry I was confused by the diff.